src/Api/Controller/ListPostsController.php

```diff
@@ -107,7 +107,7 @@ protected function extractOffset(ServerRequestInterface $request)
      */
     private function getPostIds(ServerRequestInterface $request)
     {
-        $filter = $this->extractFilter($request);
+        $filter = $this->extractFilter($request) ?: [];
```

Comment:
This is the wrong place to fix this. If you look at the `extractFilter` method the phpdoc even mentions returning an array. You should do this check over [there](https://github.com/datitisev/core/blob/4ed4067c3f01e646dddcbfe71010c476a21da10a/src/Api/Controller/AbstractSerializeController.php#L189).

Makes more sense, if you ask me 👍